docs/SPEC.md

```diff
@@ -103,7 +103,7 @@ The following keywords are reserved and may not be used as identifiers:
 
 The following character sequences represent operators:
 
-    +   ==   !=   (   )
+    +   ==   !=   (   )   =>
```

Comment:
I see what you meant on slack.  2 rows of 13 might be ok but a bit wide.  4 rows of 7/6?

Reply:
I know! They're all so nicely grouped together by function as it is! It's driving me crazy!

We can fix this later. I'm going to merge so we have a functional scanner and I'll take care of my need for nicely formatted columns later.